### ISSUE TITLE
Missing icons in PRO version 5.1.0-2

### ISSUE THREAD

**Issue by ALPIXELMaiki** [FortAwesome/Font-Awesome#13433]
Hi all,

Excuse for my english :)

I have just installed PRO version with npm, and i had a try to display icons on my project (all is working fine).
I would like to display this icon : https://fontawesome.com/icons/barcode-read?style=solid
This icon has been added in v5.0.7.

The icon is not shown. So i had a look into less files (i work with less), and in files _variables.less and _icons.less, there is no entry for my search "barcode-red" in both files.

I checked my fontawesome.less file, and i have the version 5.1.0-2.

Do you know why ?
Do you need part of my files ?
Thanks

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

> I checked my fontawesome.less file, and i have the version 5.1.0-2.

This is an early pre-release version of Font Awesome 5.1 and it may not include 5.0.7 icons (I mean it could be based on a previous version of FA 5.0).

Also, there is an issue in the Pro repository, and the 5.1.0 final release appears to not be published yet (known issue, ref: #13386)

Please upgrade at least 5.1.0-10 prerelease and check if the icons you need work

PS: The nex time, please open Font Awesome Pro related issue in the Font Awesome Pro repo at https://github.com/FortAwesome/Font-Awesome-Pro

If you don't have access to the above repository, please make sure you have linked your Font Awesome Account to your GitHub account at https://fontawesome.com/account/services

**Comment by JanKudera** [comment]
Hello, I must report same problem comparing version 4 and version 5 Pro (5.8.2 desktop OTF file for windows) that we bought. There are missing icons despite card for each image is showing  the same unicode value, e.g. following characters:
f1e0
f1c3
f15d
f160
f161
f142
f0f6
f15b
f12a
f188

I will be grateful not use two versions for same font for our hundred pages big user guide. Even more the PRO version is 1pt bigger.

**Comment by tagliala** [comment]
Hi @JanKudera 

Sorry about your upgrading issues, let's see if I can help

> I must report same problem comparing version 4 and version 5 Pro

Font Awesome 5 is not backward compatible with FA4 for every use case.

We have upgrade shims for the web, documented in our [Upgrading guide](https://fontawesome.com/how-to-use/on-the-web/setup/upgrading-from-version-4)

> I will be grateful not use two versions for same font for our hundred pages big user guide. 

If I well understand, you are trying to upgrade some file, let's say a PDF/Doc file. In that case, we do not have an upgrade script. I think it depends on the software used and we have an upgrading guide for the desktop available at this address: https://fontawesome.com/how-to-use/on-the-desktop/setup/upgrading-from-version-4

> There are missing icons despite card for each image is showing the same unicode value, e.g. following characters:

That's right. Unicode values of non-outlined icons (and outlined icons without their non-outlined match) should be the same.

What I can think is that the program is not using the proper font (font name has been changed, even from Font Awesome 5.7 to 5.8) or that the program is using by mistake the free version of the regular style. Another thing that has been changed is the font weight usage (it should be 900 for solid icons, the default one is 400). If you need icons in regular style, please double check that you are using `Font Awesome Pro Regular`.

If you put `f142` in your document (or, better, write `ellipsis-v` and take advantage of [ligatures](https://fontawesome.com/how-to-use/on-the-desktop/referencing-icons/using-ligatures)), you are supposed to see [fa-ellipsis-v in regular style](https://fontawesome.com/icons/ellipsis-v?style=regular). I can confirm it works in all three styles

![image](https://user-images.githubusercontent.com/556268/58711466-ea556280-83be-11e9-8a0f-723520a72146.png)

`f1c3` will also work because t here wasn't a non-outlined version of `fa-file-excel-o`, instead, will not work. That's because we have removed outlined icons and they are the same unicode point of the solid icon `

> Even more the PRO version is 1pt bigger.

This change has been done by design. We switched from 14px pixel size to a 16px size, partly because Bootstrap did the same for the base font size, but there are also other reasons.

Anyway... This question is related to an old pre-release of Font Awesome 5.1 and it is not the same issue as yours.

If you need more information, or you think there is an issue with Font Awesome, please start a new issue by filling out our [bug report template](https://github.com/FortAwesome/Font-Awesome/issues/new?template=bug-report.md). Remember to provide details about the os, software, and versions you are using

**Comment by JanKudera** [comment]
Hello, thank you for comprenensive answer. Let met provide you with additional information. 
I am at windows 10, using microsoft word 365, installed only one font "font awesome 5 pro-solid-900 otf." Lets say I wish insert your icon: ellipsis-v. with unicode f1c3. I type to application in regular font (e.g. calibry). I see "f881". With selected string I press alt-x and switch to awesome font. The result is a Flask with different unicode (f0c3). With previously mentioned codes above I see a) blank square b) different icon, not matching with your guide.

**Comment by tagliala** [comment]
@JanKudera try the following:

1. Uninstall all the previous versions of Font Awesome
2. Restart the system
3. Make sure that you only see the ligatures or blank squares
4. Install latest version of Font Awesome, in both regular and solid style (regular is needed if you need the old outlined icons)
5. Try again

Please also keep in mind that:
- Version 5 supports ligatures, so you should be able to write `user` and see the user icon
- Solid icons requires **bold** (or Solid font weight) to work properly: if you leave it at regular (default), I cannot guarantee that your software will display the proper icon. Take a look at the following dropdown 
![image](https://user-images.githubusercontent.com/556268/59753790-2a548a80-9285-11e9-872a-99102c80427d.png)

**Comment by JanKudera** [comment]
@tagliala 
I have not any version 4 of Awesome font anymore, from my first post, the "old" computer died. Now I have new workstation, with clean OS Windows installation. Font Awsome Regular and Solid, version 5.8.2 only present. 

It looks the problem is at Microsoft Word application. I tried the same icon at Affinity Designer, the character looks correct. (see screenshot). I was told that we faced with the same problem several years ago (I have not been at the company yet). Icons were placed to User guides as PNG files. 
So far I think this is only usable solution...nothing changed. 


![obrazek](https://user-images.githubusercontent.com/36691097/59758479-532d4d80-928e-11e9-9fbd-68a7869e6d35.png)
